doc/lua_api.txt

```diff
@@ -1535,19 +1553,22 @@ Tools define:
         * Digging times
         * Damage groups
 
-#### Full punch interval
+Full punch interval
```

Comment:
This is actually `###` a level below `Tools definition`as it describes a part of the tool definition.